package.json

```
@@ -36,6 +36,7 @@
     "@balena/es-version": "^1.0.1",
     "@balena/node-metrics-gatherer": "^6.0.3",
     "@balena/pinejs": "^14.51.0",
+    "@nicksrandall/compression": "^1.0.3",
```

I'd lean towards https://www.npmjs.com/package/compression-next as the repo it's from the "more promising" PR (whatever that means as I don't think anything will be added regardless), but more importantly to me the repo is forked from `expression` and so it's possible to see the diff between the two easily for vetting how out of date/diverged it is. It also has slightly more usage but it's minimal either way
```suggestion
    "compression-next": "^1.0.3",
```